client/extensions/woocommerce/state/sites/shipping-zone-locations/actions.js

```diff
@@ -20,7 +20,7 @@ export const fetchShippingZoneLocations = ( siteId, zoneId ) => ( dispatch, getS
 		return;
 	}
 
-	//0 is Rest of the World zone, skip fetching locations
+	//0 is Locations not covered by your other zones zone, skip fetching locations
```

Comment:
It's only changed in the UI of the Zones list. It's still called `Rest of the World` everywhere else, like in the API responses and in the "Edit Zone" screen:
![rest-of-the-world](https://user-images.githubusercontent.com/1715800/27959337-f48e8258-631e-11e7-8977-6e1c8fcb518e.png)